docs/getting-started.md

```diff
@@ -30,6 +30,12 @@ cd my-app
 yarn install
 ```
 
+3. Generate native source code:
+
+```bash
+yarn prebuild
+```
+
```

Comment:
This isn't strictly required. The first time we run `yarn ios` or `yarn android` it will be run under the hood. Well, even `yarn install` isn't needed at this point as `yarn create expo ...` does it for us too.

Anyway, I wanted to keep the first steps as short as possible. 
But builders will need to run that command if they update the framework.
I guess we should document how to update the framework and move that there.